src/code42cli/cmds/devices.py

```diff
@@ -436,6 +452,25 @@ def _add_usernames_to_device_dataframe(sdk, device_dataframe):
     return device_dataframe.merge(users_dataframe, how="left", on="userUid")
 
 
+def _add_storage_totals_to_dataframe(df, include_backup_usage):
+    df[["archiveCount", "totalStorageBytes"]] = df["backupUsage"].apply(
+        _break_backup_usage_into_total_storage
+    )
+
+    if not include_backup_usage:
+        df = df.drop("backupUsage", axis=1)
+    return df
+
+
+def _break_backup_usage_into_total_storage(backup_usage):
+    total_storage = 0
+    archive_count = 0
+    for archive in backup_usage:
```

Comment:
I wonder if we should filter out any V2 archives from these results, as it could cause confusion if 2 archives show up in the list when only 1 is visible in console. We can clarify this in the cmd help text to say `Include backup archive count and total storage in output.`